libr/core/canal.c

```diff
@@ -2280,43 +2281,65 @@ R_API void r_core_anal_coderefs(RCore *core, ut64 addr) {
 	}
 }
 
-R_API void r_core_anal_importxrefs(RCore *core) {
+static void add_single_addr_xrefs(RCore *core, ut64 addr, RGraph *graph) {
+	RFlagItem *f = r_flag_get_at (core->flags, addr, false);
+	char *me;
+	r_return_if_fail (graph);
+	me = (f && f->offset == addr)
```

Comment:
Define and assign in the same line

Reply:
I am fine with whatever style you choose, but shouldn't code style guide be updated in that case. https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md requests not to use C99 variable declarations.